_posts/2019-04-16-sicko-mode.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "\"Sicko\" Mode: a modern critique of homophobia in America"
+author: Michael Luo
+date:   2019-04-16
+categories: drake travis scott michael
+---
+
+A less careful listener may simply dismiss Travis Scott and Aubrey “Drake” Graham's hit song *Sicko Mode* as a mere modern pop trap record, perhaps noting the song’s strange choice of three different beats before requesting a change of song to Shack Wes' *Mo Bamba*. The risk of such negligent listening is to miss the song’s true message: the story of a young man exploring his sexuality. 
+
+The first beat of the song is mild and unassuming - a careful allegory chosen to represent our society’s heteronormativity. When our protagonist is in the stage this song describes (i.e. has still not come out), by obeying the heterosexual norms of society, he is repressed just like the beat of the song. Aubrey “Drake” Graham (hereby referred to as solely as Drake), plays a close friend of our protagonist. He notes that as for heterosexuality, “that shit all he know, he don’t know nothin’ else”. Unable to see our hero live a lie of a life, our artists take action by awakening his true sexuality (“i tried to show him, yea i tried to show him”). Their attempt is successful, as they note our character is “goin’ on you with the pick and roll”. For readers unfamiliar with basketball, a pick and roll involves a player forcing a defender to *switch* to defending him by setting up a screen. These words are accompanied by a beat switch - and also accompanied by our character’s switch of sexuality. 
```

Comment:
"Unable to see our hero live a lie of a life, our artists take action by awakening his true sexuality..." - the second half of this sentence is a bit confusing, as there are a lot of pronouns. I'd suggest replacing 'our artists' and 'his' with a more explicit description.

Reply:
I don't want to replace 'our artists'. One of the jokes here is that I take time to say ([Aubrey Graham] is hereby solely referred to as Drake) but I never explicitly mention him again.